### DISCUSSION TITLE
Searching Tags logic

### DISCUSSION THREAD

**Discussion by sheldon-walters** [nicolaasburgers/definition-library-releases#39] [Ideas]
Currently searching for multiple TAGS applies an AND logic -- an OR logic would be useful too  (maybe a radio button to switch between)

**Comment by nicolaasburgers** [comment]
Good idea but do you think that will lead pretty quickly to wanting a more advanced query capability where you'd want to AND some and OR others?

Or would a radio button for a global AND vs OR serve the majority of your search needs for now?

**Reply by sheldon-walters** [reply]
I reckon global radio toggle on the search logic is sufficient

**Comment by jakubczajczynski** [comment]
It's true that this feature could grow very quickly, personally I'd love to see support for regexes or pattern matching that can be saved as "saved search". Even if they can't be saved inside the DefinitionLibrary, the ability to paste a search formula that we have saved somewhere in Notepad would be cool.
